src/api/ApiClient.js

```diff
@@ -3,16 +3,44 @@ import base64 from 'base-64';
 
 const apiVersion = 'api/v1';
 
+type Account = {
```

Comment:
We might want to move this type to `user/userReducers.js` and import it here (since that's where the account object comes from)